commerce-service/src/main/java/com/liferay/commerce/internal/model/listener/CommerceCurrencyModelListener.java

```diff
@@ -36,7 +36,7 @@
 	public void onBeforeRemove(CommerceCurrency commerceCurrency) {
 		try {
 			_commercePriceListLocalService.updateCommercePriceListCurrencies(
-				commerceCurrency.getCommerceCurrencyId());
+				commerceCurrency.getCode());
```

Comment:
this change doesn't quite make sense to me. It seems like we're decoupling `CommercePriceList` and `CommerceCurrency` by relying on the currency code to link them, rather than the commerceCurrencyId. But then why do we need to touch the price list at all when a currency is removed? If we're still tying them like this, where there must be a `CommerceCurrency` for the value saved on the price list, why switch away from commerceCurrencyId at all?